scripts/release/createDataWidgetsModule.js

```diff
@@ -0,0 +1,105 @@
+const { basename, join } = require("path");
+const { readdir, copyFile, rm } = require("fs/promises");
+const { mkdir } = require("shelljs");
+const {
+    setLocalGitCredentials,
+    execShellCommand,
+    getFiles,
+    getPackageInfo,
+    createMxBuildContainer,
+    updateChangelogs,
+    githubAuthentication,
+    bumpVersionInPackageJson
+} = require("./module-automation/commons");
+
+main().catch(e => {
+    console.error(e);
+    process.exit(-1);
+});
+
+async function main() {
+    const moduleName = process.env.TAG.split("-v")[0];
+    if (moduleName !== "data-widgets") {
+        return;
+    }
+
+    await createDataWidgetsModule();
+}
+
+async function createDataWidgetsModule() {
+    console.log("Creating the Data Widgets module.");
+    const widgets = [
+        "datagrid-date-filter-web",
+        "datagrid-dropdown-filter-web",
+        "datagrid-number-filter-web",
+        "datagrid-text-filter-web",
+        "datagrid-web",
+        "dropdown-sort-web",
+        "gallery-web",
+        "tree-node-web"
+    ];
+    const DWFolder = join(process.cwd(), "packages/modules/data-widgets");
```

Comment:
Is there a reason to do `process.cwd()` everytime instead of extracting it to a const once?

Reply:
Fixed